src/fmu/sumo/sim2sumo/tables.py

```diff
@@ -156,7 +157,10 @@ def get_table(
         print("------------------")
         print(SUBMOD_DICT[submod]["doc"])
         print("------------------")
+        # TODO: see if there is a cleaner way with rft, see functions
+        # find_md_log, and complete_rft, but needs really to be fixed in res2df
     else:
+        md_log_file = find_md_log(submod, kwargs)
```

Comment:
Should we move this to inside the `if submod == rft` statement a bit further down?
No need to check for "md_log_file" unless we are working with rft?

Could also then remove
```
if submod != "rft":
    return None
```
from `find_md_log`

Reply:
No, can't do it, because we have this doorkeeper code as above:
`   output = extract_df(
                res2df.ResdataFiles(datafile_path),
                **kwargs,
            )`
Which can't handle extra arguments, but yes it needs fixing at some stage, that is why there is a TODO on this, but cant be fixed now..

Reply:
#71